libs/blocks/merch/merch.js

```diff
@@ -3,7 +3,7 @@ import {
 } from '../../utils/utils.js';
 import { replaceKey } from '../../features/placeholders.js';
 
-export const PRICE_LITERALS_URL = 'https://milo.adobe.com/libs/commerce/price-literals.json';
+export const PRICE_LITERALS_URL = 'https://www.adobe.com/federal/commerce/price-literals.json';
```

Comment:
Does business/blog use commerce? It likely makes sense to use something similar to what we do in the gnav (at some point we should also export the method properly as milo core util)

https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L13-L18
https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/utilities/utilities.js#L75-L79

Reply:
I'd only add the logic if the merch block is used on bacom/blog, otherwise duplicating the logic (for now) wouldn't be worth it

Reply:
not to my knowledge, no :)